kikuchipy/signals/ebsd.py

```diff
@@ -881,6 +883,54 @@ def get_image_quality(self, normalize: bool = True) -> np.ndarray:
 
         return image_quality_map
 
+    def index(
+        self,
+        simulations: Union["EBSD", "LazyEBSD", List[Union["EBSD", "LazyEBSD"]]],
+        metric: Union[str, SimilarityMetric] = "zncc",
+        keep_n: int = 1,
+        n_slices: int = 1,
+        merge_xmaps: bool = False,
+        osm: bool = False,
+    ) -> List[CrystalMap]:
+        """Perform Dictionary Indexing on patterns against pre-computed simulations.[ref here or elsewhere?]
```

Comment:
A reference to the Chen et al. (2015) paper should be added (does not exist in other places): http://dx.doi.org/10.1017/S1431927615000756

Reply:
This has been added.